sw/device/tock/README.md

```diff
@@ -0,0 +1,52 @@
+# Tock OS
+
+Tock OS is a secure embedded operating system for microcontrollers.
+
+This is not a complete Tock OS implementation. OpenTitan relevant contents of
```

Comment:
This could be misread as being a statement about the level of progress in porting Tock. How about keeping it shorter with something like "This directory contains OpenTitan board and chip configurations for Tock that are maintained downstream."

Reply:
now it reads as follows:
```
This directory contains OpenTitan board and chip implementation for Tock that
are maintained downstream. Other OS components are pulled directly from
upstream or a local Tock OS repository. See below for more details.
```